src/ScriptCs/Command/ExecuteLooseScriptCommand.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using ScriptCs.Contracts;
+
+namespace ScriptCs.Command
+{
+    internal class ExecuteLooseScriptCommand : IExecuteLooseScriptCommand
```

Comment:
this is almost a copy of `ExecuteScriptCommand`, the only difference appears to be `_scriptExecutor.Execute` vs `_scriptExecutor.ExecuteScript`.
It would be better to add a common `abstract class` and share most of the logic